AirRobeDemo/AirRobeDemo/Controllers/CartPageViewController.swift

```diff
@@ -20,6 +20,8 @@ final class CartPageViewController: UIViewController {
         super.viewDidLoad()
         initCart()
         initMultiOptIn()
+
+        emailTextField.delegate = self
```

Comment:
the library didn't do the email validation itself, and I believed it can be taken care of by the consumer.
But for sure, I can add email validation in another PR.
Thanks for pointing out this issue. 👍

Reply:
I am not sure it's the right thing to do, which is why I posed it as a question. For example, is the customer OK if we block the checkout flow due to an invalid email address for the AirRobe opt-in? Maybe check with Victor or whoever is the stakeholder for this aspect of this before acting on my question. :)

Reply:
Cc @michaeldawson @piusj